File: exercises-python/3. Repetitivos/17.py
'''
17.Desarrolle los métodos que implementen los métodos IndexOf de la clase String.
'''

import os
os.system("cls")

texto = input("Texto: ")
busca = input("Buscar: ")
contador = 1

for i in texto:
    if busca == i:
        break
    else:
        contador += 1
if contador == len(texto) + 1:print("No se encontró") 
else: print(f"Está en la posición {contador}")

File: exercises-cpp/3. Repetitivas/17.cpp
#include<iostream>
#include<string.h>
using namespace std;

int main(){
    system("cls");
    string texto,busca;
    int posicion;
    cout<<"Texto: "; cin>>texto;
    cout<<"Buscar: "; cin>>busca;
    posicion = texto.find(busca);

    cout<<"Esta en la posicion: "<<posicion+1;
    return 0;
}